python/vivado.py

```diff
@@ -99,7 +105,7 @@ def xilinx_ultrascale_plus_vivado_synthesis(
 
 read_verilog -sv ${{sv_source_file}}
 set_property top ${{modname}} [current_fileset]
-{synth_design_command if synth_design else f"# {synth_design_command}"}
+{synth_design_command if synth_design else f"# {synth_design_command}"} 
```

Comment:
i think you need to run the `fmt.sh` script, looks like you ahve some formatting bugs

Reply:
Ahh I see, thanks!